ding/ding.py

```diff
@@ -82,10 +82,15 @@ def print_time(seconds):
     """Print countdown for `seconds`"""
     while seconds > 0:
         start = time.time()
-        os.system('cls' if os.name == 'nt' else 'clear') # accommodate Windows
-        print(datetime.timedelta(seconds=seconds))
+        if os.name == 'nt':
```

Comment:
I just tested on Windows, and it seems like this works there too! Therefore, we don't need any special handling besides the initial `clear` / `cls`.